README.md

```diff
@@ -117,101 +117,81 @@ On your host system create two secure (meaning not world-readable) files contain
 1. ib.account.txt
 2. ib.password.txt
 
-Next, inject these secrets into the Docker Swarm by using `docker secret create`:
+Next, inject these secrets into the Docker Swarm by using the `docker secret create` command.
 
 ```posh
 docker secret create IBEAM_ACCOUNT_v1 ib.account.txt
 docker secret create IBEAM_PASSWORD_v1 ib.password.txt
 ```
 Once you've initialized the secrets delete the original files from your host system.
 
-In Docker Swarm mode, in order to enable IP-based access control for the IBeam service, we need to create an overlay network in Docker:
-
-```posh
-docker network create --driver overlay --attachable ib_net_01
-```
-
-Next, create an [inputs][inputs-and-outputs] directory with a `conf.yaml` file.  The format of this file is discussed on the the [Gateway Configuration][gateway-configuration] page.
+Next, create an [Inputs Directory][inputs-and-outputs] with a `conf.yaml` file.  The format of this file is discussed on the the [Gateway Configuration][gateway-configuration] page.
 Toward the end of the `conf.yaml` there is a block to define IPs to trust and reject, e.g.,
 
 ```yaml
 ...
 ips:
   allow:
     - 127.0.0.1
-    - 172.18.0.1
   deny:
     - 0-255.*.*.*
 ```
+The example above grants access from the local loopback interface, `127.0.0.1`, and denies all other addresses (`0-255.*.*.*`).
 
-The example above grants access from the container's local interface and from the gateway IP address used to route traffic from the local host to the container.
-
-When accessed from the local host Docker Swarm will route traffic over a gateway interface, `docker_gwbridge`, that it sets up.
-
-```posh
-docker network inspect docker_gwbridge
-...
-        "IPAM": {
-            "Driver": "default",
-            "Options": null,
-            "Config": [
-                {
-                    "Subnet": "172.18.0.0/16",
-                    "Gateway": "172.18.0.1"
-                }
-            ]
-        },
-...
-```
-
-The loopback address, 127.0.0.1, and the gateway address listed in the example above, `172.18.0.1`, are granted access in our example `conf.yaml`.
-All other IPs, `0-255.*.*.*` in our example `conf.yaml`, are denied access.
-
-To deploy IBeam as a service named 'ibeam' from the command line:
+To deploy IBeam as a service named 'ibeam' we will use the `docker service create` command.
 
 ```posh
 docker service create \
     --name ibeam \
-    --network ib_net_01 \
+    --network host \
```

Comment:
Isn't this the default network? Do we need to specify it?

Reply:
Under Docker Swarm the default network will be the 'bridge' network, which hides the actual incoming IP address from the service.